### ISSUE TITLE
Bitmex api wrong OHLCV data

### ISSUE THREAD

**Issue by timofeyAntonenkoJelvix** [BitMEX/api-connectors#261]
Example of a request:
https://testnet.bitmex.com/api/v1/trade/bucketed?binSize=1m&partial=false&symbol=XBTUSD&count=100&reverse=false&startTime=2018-05-05T04%3A02%3A00.000Z

Take a look at timestamp "2018-05-05T04:52:00.000Z".
Open price "9853" is lower than Low price "9854".

**Comment by Madward** [comment]
Yes, on both testnet and mainnet some candles have their "low" value above the "open" value.

The open value is right, the low value is wrong. Same for close and high. Given the fact BitMEX will most likely not fix this, you should do it yourself:

`For each candle in candles
    if (max(candle.open, candle.close) > candle.high)
        candle.high=max(candle.open, candle.close)
    else if (min(candle.open, candle.close) < candle.low)
        candle.low=min(candle.open, candle.close)`

Regards.

**Comment by timofeyAntonenkoJelvix** [comment]
@Madward Thanks a lot!

**Comment by Luther-Varys** [comment]
I've recently tried the fix but I'm still get incorrect prices
